PostFreeSurfer/PostFreeSurferPipeline_1res.sh

```diff
@@ -180,7 +180,14 @@ if [ `${FSLDIR}/bin/imtest ${T2wFolder}/T2w` -eq 0 ]; then
 fi
 
 if [ "${RegName}" = "FS" ] ; then
-  logWarn "WARNING: FreeSurfer registration is deprecated in the HCP Pipelines as it results in poorer cross-subject functional and cortical areal alignment relative to MSMSulc. Additionally, FreeSurfer registration results in dramatically higher surface distortion (both isotropic and anisotropic). These things occur because FreeSurfer's registration has too little regularizattion of folding patterns that are imperfectly correlated with function and cortical areas, resulting in overfitting of folding patterns. See Robinson et al 2014, 2018 Neuroimage, and Coalson et al 2018 PNAS for more details."
+    log_Warn "WARNING: FreeSurfer's surface registration (based on cortical folding) is deprecated in the"
```

Comment:
Same, "WARNING" is redundant.

Reply:
Also, update to match the multi-line version.

Reply:
Changed